website/content/docs/agent-and-proxy/agent/template.mdx

```diff
@@ -301,47 +301,54 @@ behaviors on certificates:
 
 The following demonstrates Vault Agent Templates configuration blocks.
 
-```hcl
-# Other Vault Agent configuration blocks
-# ...
+<Tabs>
+  <Tab heading="File Templates">
+    ```hcl
```

Comment:
You need a line between the tab definition and the codeblock for them to render properly! See a note on this in the [source documentation here](https://github.com/hashicorp/vault/blob/main/website/README.md#tabs).